web/app.js

```diff
@@ -1297,38 +1306,113 @@ var PDFViewerApplication = {
     }
   },
 
-  bindWindowEvents: function pdfViewBindWindowEvents() {
-    var eventBus = this.eventBus;
+  bindWindowEvents() {
+    let eventBus = this.eventBus;
+
+    // Window Resize
+    this._boundEvents.windowResize = (function () {
+      this.dispatch('resize');
+    }).bind(eventBus);
```

Comment:
Please don't use `bind` needlessly; I think the following ought to work instead:
```javascript
this._boundEvents.windowResize = () => {
  eventListener.dispatch('resize');
};
```